Tools/ApplicationDebugger/jacobi/jacobi-fixed.vcxproj.user

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+    <LocalDebuggerCommandArguments>cpu</LocalDebuggerCommandArguments>
+    <DebuggerFlavor>WindowsLocalDebugger</DebuggerFlavor>
+    <RemoteDebuggerCommandArguments>gpu</RemoteDebuggerCommandArguments>
+    <LocalDebuggerEnvironment>CL_CONFIG_USE_NATIVE_DEBUGGER=1</LocalDebuggerEnvironment>
```

Comment:
Should this be defined for the remote env, too?

Reply:
I would say no, as we use remote for GPU most of the time.